pages/ssj/[workflow]/index.js

```diff
@@ -943,15 +928,6 @@ const AddOpenDateModal = ({ toggle, open, openDate, setOpenDate, team }) => {
     </Modal>
   );
 };
-const ViewEtlsModal = ({ toggle, open, etls }) => {
-  return (
-    <Modal title="Meet your peers" toggle={toggle} open={open}>
-      <Stack spacing={3}>
-        <ETLs etls={etls} />
-      </Stack>
-    </Modal>
-  );
-};
```

Comment:
https://trello.com/c/wwN9za5z/948-hide-the-other-etls-section-on-the-ssj-index?search_id=8987377f-e689-4311-b36c-1c3836a6e30f

Reply:
Same as above for here.. I'd leave this and not remove it